### PR TITLE
[Remove usages of Stream:asset]Lps 170480

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-170480

### CODE REVIEW

File: modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java [hunk 1]
```diff
@@ -144,30 +143,32 @@ public String getActualURL(
 			infoItemFieldValuesProvider.getInfoItemFieldValues(
 				layoutDisplayPageObjectProvider.getDisplayObject());
 
-		Optional<String> descriptionOptional = _getMappedValueOptional(
+		String description = _getMappedValueOptional(
```
Comment (dantewang) [inline modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java:146]: @slnn Please remember to rename the method.

Comment (slnn) [reply modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java]: Sorry, my bad. Forced push with updating.

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:stop

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/128338

**Pull request closed without merging**